common/dao-api/src/main/java/org/thingsboard/server/dao/cassandra/AbstractCassandraCluster.java

```diff
@@ -36,6 +36,8 @@
     private Boolean jmx;
     @Value("${cassandra.metrics}")
     private Boolean metrics;
+    @Value("${cassandra.local_datacenter}")
```

Comment:
please add default value here. This will make sure that everything will work for old clients, even if they use same thingsboard.yml